### PR TITLE
TFT Display for Switchwire + Afterburner mount for Input shaper (also works for Y on SW)

### PR DESCRIPTION
**Pull request by thiagolocatelli**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

File: printer_mods/DoubleT/input_shaper_mount/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Input shaper mount for Afterburner
+
+This mode replaces the front middle skirt in order to mount a 3.5in touchscreen display for Klipperscreen. It has two printed parts: the main body and the front plate. Threaded inserts go into the main body and the M3x8 screws are used to mount the front plate onto the main body.
```
Comment (KiloQubit) [inline printer_mods/DoubleT/input_shaper_mount/readme.md:3]: I think you got the two readmes mixed up, this is talking about the display mount in the input shaper mount mod.

File: printer_mods/README.md [hunk 2]
```diff
@@ -82,6 +82,8 @@ like so:
 | |[Duet 3 3HC Din Bracket](./DorjeDorf/Duet_3_3HC_Din_Bracket) |  V2.4 Duet 3 3HC Din Bracket | :x: | :x: | :heavy_check_mark: |:x:|
 |DoubleT |[Afterburner Triple Bowden](./DoubleT/afterburner_triple_bowden) |AfterBurner Triple Bowden|:x:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
 | |[Spider Mount Extrusion](./DoubleT/fysetc_spider_mount_extrusion) |Spider Mount for 2020/3030 extrusions|:x:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+| |[TFT Display skirt for Switchwire](./DoubleT/switchwire_tft_display) |Spider Mount for 2020/3030 extrusions|:x:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
```
Comment (KiloQubit) [inline printer_mods/README.md:85]: The description column for both of these mods here is copied from your previous mod.

File: printer_mods/DoubleT/input_shaper_mount/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+![image1](./images/image1.png)
+
+## BOM
+
+- 2 Threaded inserts (stock)
+- 2 M3x16 BHCS or SHCS
+- 2 M3x6 BHCS
+- 2 round magnets 8x4
+- Input Shaper [ADXL345](https://www.amazon.com/gp/product/B07DMZCGP9/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)
```
Comment (KiloQubit) [inline printer_mods/DoubleT/input_shaper_mount/readme.md:13]: Please remove the extraneous things from the URL, it should be just

    https://www.amazon.com/gp/product/B07DMZCGP9

File: printer_mods/DoubleT/input_shaper_mount/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Input shaper mount for Afterburner
+
+This mode replaces the front middle skirt in order to mount a 3.5in touchscreen display for Klipperscreen. It has two printed parts: the main body and the front plate. Threaded inserts go into the main body and the M3x8 screws are used to mount the front plate onto the main body.
+
+![image1](./images/image1.png)
+
```
Comment (KiloQubit) [inline printer_mods/DoubleT/input_shaper_mount/readme.md:6]: Where does this mount on the Afterburner and what are the magnets and heatset inserts for?

File: printer_mods/DoubleT/switchwire_tft_display/readme.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# 3.5in TFT Display Front Skirt - Switchwire
+
+This mode replaces the front middle skirt in order to mount a 3.5in touchscreen display for Klipperscreen. It has two printed parts: the main body and the front plate. Threaded inserts go into the main body and the M3x8 screws are used to mount the front plate onto the main body.
```
Comment (KiloQubit) [inline printer_mods/DoubleT/switchwire_tft_display/readme.md:3]: Replace `mode` with `mod`.

File: printer_mods/DoubleT/switchwire_tft_display/readme.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+![image1](./images/image1.png)
+
+![image2](./images/image2.png)
+
+This is a remix of the original display mount by `rcfishhunt`.
```
Comment (KiloQubit) [inline printer_mods/DoubleT/switchwire_tft_display/readme.md:9]: Is the original mount somewhere in github?  Can you add a link?

File: printer_mods/DoubleT/switchwire_tft_display/readme.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+## BOM
+
+- 5 threaded inserts (stock)
+- 4 M3x8 BHCS
+- [3.5in TFT Display](https://www.amazon.com/gp/product/B07KLDKM5D/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)
```
Comment (KiloQubit) [inline printer_mods/DoubleT/switchwire_tft_display/readme.md:15]: Please remove extraneous stuff from the URL, should be just

    https://www.amazon.com/gp/product/B07KLDKM5D

File: printer_mods/DoubleT/switchwire_tft_display/readme.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+Follow this [tutorial](https://github.com/jordanruthe/KlipperScreen/blob/master/docs/Installation.md) to install Klipperscreen on your Raspberry Pi.
+
+### LCD Show
+
+Ssh into your Raspberry Pi and execute the following commands
+
+```
+sudo rm -rf LCD-show
+git clone https://github.com/Lcdwiki/LCD-show.git
+hmod -R 755 LCD-show
```
Comment (KiloQubit) [inline printer_mods/DoubleT/switchwire_tft_display/readme.md:32]: Change `hmod` to `chmod`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

`switchwire_tft_display/CAD/switchwire_tft_display_mount.f3z.f3d` has a weird filename - it is an f3z or f3d ?

See other changes below.

**Comment by thiagolocatelli** [comment]
@KiloQubit changes applied

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**